### PR TITLE
RHDEVDOCS-3616 ODC Release Notes, Bug Fixes for 4.10

### PR DESCRIPTION
**Pull request by rolfedh**
- Aligned team: Dev Tools
- For branches: enterprise-4.10 release notes
- Jira: https://issues.redhat.com/browse/RHDEVDOCS-3616
- Direct link to doc preview: https://deploy-preview-42159--osdocs.netlify.app/openshift-enterprise/latest/release_notes/ocp-4-10-release-notes#ocp-4-10-web-console-developer-perspective-bug-fixes
- SME review: Received in the individual BZs.
- QE review: @sanketpathak 
- Peer review: @jc-berger 
- <All reviews complete. Please merge now>

### CODE REVIEW

File: release_notes/ocp-4-10-release-notes.adoc [hunk 1]
```diff
@@ -799,6 +799,28 @@ You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8. Yo
...
+
+* Before this update, the name input fields used a complex and recursive regular expression to validate user inputs. This regular expression made name detection very slow and often caused errors. This update resolves the issue by optimizing the regular expression and avoiding recursive matching. Now, name detection is fast and does not cause errors. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2014497[BZ#2014497])
+
+* Before this update, feature flag gating was missing from some extensions contributed by the knative plugin. Although this issue didn't affect what was displayed, these extensions ran unnecessarily, even if the serverless operator was not installed. This update resolves the issue by adding feature flag gating to the extensions where it was missing. Now, the extensions do not run unnecessarily. (link:https://bugzilla.redhat.com/show_bug.cgi?id=2016438[BZ#2016438])
```
Comment (jc-berger) [inline release_notes/ocp-4-10-release-notes.adoc:812]: Second sentence contraction, `didn't`. I don't think the contraction is bad here. However, I don't notice many contractions throughout these web console updates. For consistency, I would simply avoid using contractions unless you do so frequently throughout the updates.

Comment (rolfedh) [reply release_notes/ocp-4-10-release-notes.adoc:812]: Updated. Thanks.

**Review by jc-berger — APPROVED** [review]
/lgtm

**Pull request merged**